android/java/res/layout/fragment_unlock_wallet.xml

```diff
@@ -1,119 +1,96 @@
-                android:paddingBottom="10dp"
-                android:textColor="@color/wallet_text_color"
-                android:textColorHint="@color/brave_theme_hint_text"
-                android:textCursorDrawable="@null"
-                android:textSize="14sp"
-                android:visibility="gone" />
+    </com.google.android.material.textfield.TextInputLayout>
 
-            <ImageView
-                android:id="@+id/iv_biometric_unlock_wallet"
-                android:layout_width="48dp"
-                android:layout_height="match_parent"
-                android:layout_marginStart="2dp"
-                android:contentDescription="@string/accessibility_biometric_unlock"
-                android:padding="6dp"
-                android:visibility="gone"
-                app:srcCompat="@drawable/ic_fingerprint"
-                app:tint="@color/brave_action_color" />
-
-        </LinearLayout>
-
-        <androidx.appcompat.widget.AppCompatButton
-            android:id="@+id/btn_unlock"
-            android:layout_width="match_parent"
-            android:layout_gravity="center_horizontal"
-            android:text="@string/unlock"
-            android:layout_marginHorizontal="16dp"
-            android:layout_marginBottom="16dp"
-            android:visibility="gone"
-            style="@style/BraveWalletOnboardingButton"/>
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/btn_unlock"
+        style="@style/BraveWalletOnboardingButton"
+        android:layout_width="match_parent"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginBottom="28dp"
+        android:text="@string/unlock" />
 
-        <TextView
-            android:id="@+id/btn_unlock_wallet_restore"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:padding="8dp"
-            android:background="?android:attr/selectableItemBackground"
-            android:layout_gravity="center_horizontal"
-            android:clickable="true"
-            android:focusable="true"
-            android:textAllCaps="false"
-            android:layout_marginBottom="16dp"
-            android:text="@string/restore"
-            android:textSize="16sp"
-            android:visibility="gone"
-            android:textColor="@color/wallet_text_color"/>
+    <TextView
+        android:id="@+id/btn_unlock_wallet_restore"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginBottom="40dp"
+        android:background="?android:attr/selectableItemBackground"
+        android:clickable="true"
+        android:focusable="true"
+        android:padding="16dp"
+        android:text="@string/restore"
+        android:textColor="@color/brave_wallet_dn_button_color"
+        android:textStyle="bold"
```

Comment:
we can use styling from nala here.

Reply:
@deeppandya, where can I find a comprehensive list of the Nala styles available? 
Which is the one that matches this style? The restore button is a plain text made clickable, I'm attaching a screenshot.

<img src="https://github.com/brave/brave-core/assets/3308503/d3eaea2c-b10f-4675-884b-5a91adb0d4e2" width="300" />

Reply:
UPDATE: I've included the predefined text style `DefaultRegular` that sets some extra details.